src/main/java/org/sagebionetworks/web/client/presenter/TrustCenterPresenter.java

```diff
@@ -0,0 +1,126 @@
+
+import com.google.gwt.activity.shared.AbstractActivity;
+import com.google.gwt.event.shared.EventBus;
+import com.google.gwt.http.client.Request;
+import com.google.gwt.http.client.RequestBuilder;
+import com.google.gwt.http.client.RequestCallback;
+import com.google.gwt.http.client.Response;
+import com.google.gwt.user.client.ui.AcceptsOneWidget;
+import com.google.inject.Inject;
+import java.util.HashMap;
+import java.util.Map;
+import org.sagebionetworks.web.client.RequestBuilderWrapper;
+import org.sagebionetworks.web.client.place.TrustCenterPlace;
+import org.sagebionetworks.web.client.view.MapView;
+import org.sagebionetworks.web.client.view.SynapseStandaloneWikiView;
+import org.sagebionetworks.web.client.widget.entity.controller.SynapseAlert;
+import org.sagebionetworks.web.shared.WebConstants;
+
+public class TrustCenterPresenter
+  extends AbstractActivity
+  implements
+    Presenter<org.sagebionetworks.web.client.place.TrustCenterPlace>,
+    MapView.Presenter {
+
+  SynapseStandaloneWikiView view;
+  RequestBuilderWrapper requestBuilder;
+  SynapseAlert synAlert;
+
+  private static Map<String, String> documentKeyToGithubUrl = new HashMap<>();
+
+  static {
+    documentKeyToGithubUrl.put(
+      TrustCenterPlace.TERMS_OF_SERVICE_KEY,
+      "https://raw.githubusercontent.com/Sage-Bionetworks/Sage-Governance-Documents/main/Terms.md"
+    );
+    documentKeyToGithubUrl.put(
+      TrustCenterPlace.PRIVACY_POLICY_KEY,
+      "https://raw.githubusercontent.com/Sage-Bionetworks/Sage-Governance-Documents/main/privacy.md"
+    );
+    documentKeyToGithubUrl.put(
+      TrustCenterPlace.COOKIES_KEY,
+      "https://raw.githubusercontent.com/Sage-Bionetworks/Sage-Governance-Documents/main/cookies.md"
+    );
+    documentKeyToGithubUrl.put(
+      TrustCenterPlace.SUBPROCESSORS_KEY,
+      "https://raw.githubusercontent.com/Sage-Bionetworks/Sage-Governance-Documents/main/subprocessors.md"
+    );
+  }
+
+  @Inject
+  public TrustCenterPresenter(
+    SynapseStandaloneWikiView view,
+    RequestBuilderWrapper requestBuilder,
+    SynapseAlert synAlert
+  ) {
+    this.view = view;
+    this.requestBuilder = requestBuilder;
+    this.synAlert = synAlert;
+    view.setSynAlert(synAlert);
+  }
+
+  @Override
+  public void start(AcceptsOneWidget panel, EventBus eventBus) {
+    panel.setWidget(view.asWidget());
+  }
+
+  @Override
+  public void setPlace(
+    org.sagebionetworks.web.client.place.TrustCenterPlace place
+  ) {
+    synAlert.clear();
+    String documentKey = place.getDocumentKey();
+    String documentUrl = documentKeyToGithubUrl.get(documentKey);
+    if (documentUrl == null) {
+      synAlert.showError("Unrecognized document key: " + documentKey);
+    } else {
+      showFileContent(documentUrl);
+    }
+  }
+
+  public void showFileContent(String url) {
+    requestBuilder.configure(RequestBuilder.GET, url);
+    requestBuilder.setHeader(
+      WebConstants.CONTENT_TYPE,
+      WebConstants.TEXT_PLAIN_CHARSET_UTF8
+    );
```

Comment:
I doubt that we will have many users viewing this page, so it's unlikely we will be throttled, but I would consider moving this to the servlet and caching it.

Reply:
An older (about 3 years old) question about this topic suggests that they allow may impose a limit of 5000 requests per hour, per IP address (which would be the client IP).  
https://stackoverflow.com/questions/66522261/does-github-rate-limit-access-to-public-raw-files